igvc/src/imu/main.cpp

```diff
@@ -91,29 +99,56 @@ int main(int argc, char** argv)
 
     try
     {
-      msg.linear_acceleration.x = stof(accelTokens[0]);
-      msg.linear_acceleration.y = stof(accelTokens[1]);
-      msg.linear_acceleration.z = stof(accelTokens[2]);
+      double cur_x = stof(accelTokens[0]);
+      double cur_y = stof(accelTokens[1]);
+      double cur_z = stof(accelTokens[2]);
       msg.linear_acceleration_covariance = { 0.04, 1e-6, 1e-6, 1e-6, 0.04, 1e-6, 1e-6, 1e-6, 0.04 };
 
+      int num = 500;
```

Comment:
It looks like this variable is setting a threshold to form some kind of buffer / moving average for the x/y/z accel readings.  If that's the case, it should have a name that's more descriptive than `num`.